### PR TITLE
Add tenere

### PR DESCRIPTION
**Pull request by pythops**

### CODE REVIEW

**Comment by rothgar** [comment]
Thanks for the PR. Can you please put tenere in alphabetical order in its section?

File: README.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ List of projects that provide terminal user interfaces
 - [vlc](https://github.com/videolan/vlc) VLC includes an ncurses interface, `vlc --intf ncurses`. See documentation at https://wiki.videolan.org/Console/
 - [ytui-music](https://github.com/sudipghimire533/ytui-music) Listen to music from youtube. Configurable, minimal, lightweight, private & beautiful music client.
 - [ytdl-tui](https://github.com/darky/ytdl-tui) TUI for downloading Youtube videos
-- [ytfzf](https://github.com/pystardust/ytfzf) A POSIX script that helps you find Youtube videos (without API) or Peertube videos and opens/downloads them using mpv/youtube-dl 
+- [ytfzf](https://github.com/pystardust/ytfzf) A POSIX script that helps you find Youtube videos (without API) or Peertube videos and opens/downloads them using mpv/youtube-dl
```
Comment (jserv) [inline README.md:294]: You should keep this line unchanged to focus on reviewing the code.

Comment (pythops) [reply README.md]: Done.

**Comment by pythops** [comment]
@rothgar 
Done.

File: README.md [hunk 2]
```diff
@@ -320,6 +320,7 @@ List of projects that provide terminal user interfaces
 - [sc-im](https://github.com/andmarti1424/sc-im) An ncurses spreadsheet program for terminal. Reignited version of sc
 - [taskline](https://github.com/perryrh0dan/taskline) Tasks, boards & notes for the command-line habitat
 - [taskwarrior-tui](https://github.com/kdheepak/taskwarrior-tui) A Terminal User Interface for Taskwarrior
+- [tenere](https://github.com/pythops/tenere) A Terminal user interface (TUI) for ChatGPT written in Rust.
```
Comment (jserv) [inline README.md:323]: How about referring to it as "TUI" instead of "Terminal User Interface"?

**Review by jserv — CHANGES_REQUESTED** [review]
Execute `git rebase -i` to squash the commits, making the review and merge process more convenient.

**Pull request merged**